dipy/segment/tissue.py

```diff
@@ -125,3 +125,98 @@ def classify(self, image, nclasses, beta, *, tolerance=1e-05, max_iter=100):
         PVE = PVE[..., 1:]
 
         return initial_segmentation, final_segmentation, PVE
+
+
+def compute_directional_average(data, bvals, b0_thresh=10):
```

Comment:
- `b0_threshold=50` instead - to be consistent with the codebase
- Also, we use keyword only arguments so the definition should be: 
   `def compute_directional_average(data, bvals, *, b0_threshold=50):`